prebuild/action.yml

```
@@ -1,6 +1,8 @@
 runs:
   using: composite
   steps:
+    # In some cases (as in dd-native-appsec), it is necessary to download the artifacts previously uploaded by the parent workflow so that they can later be used in binding.gyp
```

```suggestion
    # Make artifacts previously uploaded by the parent workflow available to binding.gyp
```

Reworded more generically and also so that it's not tied to Datadog projects.